ui/src/app/pages/homepage/recent-resources.tsx

```diff
@@ -185,43 +193,61 @@ export const RecentResources = fp.flow(withCdrVersions())((props: Props) => {
     }
   }, [resources, wsMap]);
 
-  return resources && wsMap && !loading ? (
-    <React.Fragment>
-      <SmallHeader>Recently Accessed Items</SmallHeader>
-      <div data-test-id='recent-resources-table'>
-        <DataTable
-          value={tableData}
-          scrollable={true}
-          paginator={true}
-          paginatorTemplate='CurrentPageReport'
-          currentPageReportTemplate='Showing {totalRecords} most recent items'
-        >
-          <Column field='menu' style={styles.menu} />
-          <Column
-            field='resourceType'
-            header='Item type'
-            style={styles.typeColumn}
-          />
-          <Column field='resourceName' header='Name' style={styles.column} />
-          <Column
-            field='workspaceName'
-            header='Workspace name'
-            style={styles.column}
-          />
-          <Column
-            field='formattedLastModified'
-            header='Last changed'
-            style={styles.column}
-          />
-          <Column
-            field='cdrVersionName'
-            header='Dataset'
-            style={styles.column}
-          />
-        </DataTable>
-      </div>
-    </React.Fragment>
-  ) : (
-    <SpinnerOverlay />
+  return cond(
+    [
+      apiLoadError && !loading,
+      () => (
+        <React.Fragment>
+          <AlertWarning style={{ fontSize: 14 }}>
+            <ClrIcon className='is-solid' shape='exclamation-triangle' />
+            {apiLoadError}
+          </AlertWarning>
+        </React.Fragment>
+      ),
+    ],
+    () =>
+      resources && wsMap && !loading ? (
```

Comment:
this could be resources && wsMap && !loading, () =-> .....
and then check other comment,
loading, () => <SpinnerOverlay/> could be another condition

Reply:
Updated to use `cond`.